restdocs-api-spec-openapi3-generator/src/test/kotlin/com/epages/restdocs/apispec/openapi3/OpenApi3GeneratorTest.kt

```diff
@@ -30,6 +30,16 @@ class OpenApi3GeneratorTest {
     lateinit var openApiSpecJsonString: String
     lateinit var openApiJsonPathContext: DocumentContext
 
+    @Test
+    fun `should convert multi level schema medel to openapi`() {
+        givenPutProductResourceModel()
+        whenOpenApiObjectGenerated()
```

Comment:
Please introduce blank lines around this line to visually separate the given-when-then blocks.